alerting/src/main/kotlin/org/opensearch/alerting/model/WorkflowRunResult.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.alerting.model
+
+import org.opensearch.common.io.stream.StreamInput
+import org.opensearch.common.io.stream.StreamOutput
+import org.opensearch.common.io.stream.Writeable
+import org.opensearch.core.xcontent.ToXContent
+import org.opensearch.core.xcontent.XContentBuilder
+import java.io.IOException
+import java.lang.Exception
+import java.time.Instant
+
+data class WorkflowRunResult(
+    val workflowRunResult: List<MonitorRunResult<*>> = mutableListOf(),
+    val executionStartTime: Instant,
+    val executionEndTime: Instant? = null,
+    val executionId: String,
+    val error: Exception? = null
```

Comment:
should we make this a list?

Reply:
Well I think we discussed to remember only the last error if happened. Also, bear in mind that the 
  val workflowRunResult: List<MonitorRunResult<*>> = mutableListOf(),
can contain the errors if the error happens